openmetadata-docs/ingestion/connectors/postgres/usage.yaml

```diff
@@ -0,0 +1,31 @@
+source:
+  type: postgres-usage
+  serviceName: "<service name>"
+  serviceConnection:
+    config:
+      type: Postgres
+      username: openmetadata_user
+      password: openmetadata_password
+      hostPort: localhost:5432
+      database: pagila
+  sourceConfig:
+    config:
+      queryLogDuration: "<query log duration integer>"
+      stageFileLocation: "<path to store the stage file>"
+      resultLimit: "<query log limit integer>"
+processor:
+  type: query-parser
+  config:
+    filter: ''
```

Comment:
I believe this `filter` config is deprecated. We need to update the docs/examples with
```
processor:
  type: query-parser
  config: {}
```